### DISCUSSION TITLE
Analog SDR Trunk TGs / Format for CSV file for TG importation?

### DISCUSSION THREAD

**Discussion by cfc62** [chuot/rdio-scanner#98] [General]
I've made a completely analog SDRT system and tagged each TG manually. I know I'm a bit off the beaten path here but it's working pretty well with the NBFM squelch that's in beta. 

While I see that Frequency is coming across, none of the other MP3 metadata/tags are coming across. As it stands right now I'm just getting numbers for talk groups. So if my "talkgroup" for a particular frequency 154.130 is 14, then I get 14 displayed as the Tag, Description and TGID, but 154 130 000 does get displayed correctly. 

I did all of the tagging in SDRT from a spreadsheet, so I figured I could use the CSV importation function easily as I already had all the data at hand. 

I set one up with Id, Label, Description, Tag, Group as shown in the Admin section for importation.

But the fields aren't matching up when I do a trial import. 

If there's a file spec for the CSV file, I could use a pointer to it.  

Or if I've somehow botched up my SDRT tagging, that would be good to know as well, but it looks and operates as expected there.

**Comment by cfc62** [comment]
This is what I'm currently getting...

<img width="624" alt="image" src="https://user-images.githubusercontent.com/70529418/149689787-8c94bbbe-ab88-421c-a170-03d35a4fcf37.png">

<img width="1200" alt="image" src="https://user-images.githubusercontent.com/70529418/149689836-95c66b1c-445b-4af3-9de1-dd1612448b0d.png">

**Reply by chuot** [reply]
Send me 2 or 3 original mp3 files by email, I will make sure that all the mp3 tag info is being taking care of. Actually, this should have been working while autopopulate is enabled. Plase confirm you're using SDRTrunk 0.5.0 alpha 6.

**Reply by cfc62** [reply]
Thanks - I'll send a few on shortly. I can see there's metadata in the MP3s.

Autopopulate is indeed enabled, and I'm on a beta branch of SDRT (needed for NBFM squelch) which is actually newer than 0.5.0 a6. I don't see any notes that Denny has changed anything with tagging or metadata or anything like that, but it's possible.

**Comment by cfc62** [comment]
Email sent.

**Reply by chuot** [reply]
Please send me by email a screenshot of your alias table on SDRTrunk. Thanks

**Comment by chuot** [comment]
Ok, starting from v6.1.4, you will see this instead of just the talkgroup id. Note that if you change the label on the SDRTrunk side, it will also update on Rdio Scanner so you won't have to change the label at both places (sdrt & rdio). This works only if autoPopulate is enabled.

![tglabel](https://user-images.githubusercontent.com/12464613/149824459-9e4c6de8-6535-4a1d-be29-d4d87d17d9ee.png)

**Reply by cfc62** [reply]
Perfect, thanks!

**Comment by cfc62** [comment]
Looking good in 6.1.4, thanks again!
